drake/systems/plants/RigidBodyManipulator.cpp

```diff
@@ -533,7 +533,7 @@ void RigidBodyManipulator::updateCompositeRigidBodyInertias(KinematicsCache<Scal
       }
     }
 
-    for (int i = bodies.size() - 1; i >= 0; i--) {
+    for (int i = static_cast<int>(bodies.size()) - 1; i >= 0; i--) {
```

Comment:
I've started using ptrdiff_t, which is supposed to store the difference between size_t types, instead of having static_casts everywhere.